### PR TITLE
DOC: add description of the resting-state naturalistic movie

### PR DESCRIPTION
**Pull request by celprov**

### CODE REVIEW

**Comment by celprov** [comment]
@oesteban I'm sure you know the movie by heart :P, so feel free to add/modify any relevant details.

File: README.md [hunk 1]
```diff
@@ -48,7 +48,9 @@ The units reported here correspond to *Psychopy*'s "normalized units".
...
+Resting-state fMRI was acquired while the participant watched a naturalistic scene depicting a calm landscape at sunset. The movie was captured by a static camera offering the view of the village of Mundaka, located on the northern coast of Spain, as shown in the image below. Filmed in real-time, the movie lasts exactly 20 minutes and has not been accelerated, decelerated, or edited, maintaining the natural progression of the scene. As the sun sets, the luminosity gradually decreases. The movie contains no sudden or abrupt stimuli, aside from occasional passing cars in the village. Slow and steady movement visible throughout are the waves, the lights flickering and the clouds drifting. The movie is silent.
```
Comment (oesteban) [inline README.md:51]:
```suggestion
Resting-state fMRI was acquired while the participant watched a naturalistic scene depicting a calm landscape at sunset.
The movie was captured by a static camera offering the view of the village of Mundaka, located on the northern coast of Spain, as shown in the image below.
From the original recording of over four hours in length, we extracted 40 minutes, and increased the speed to compress them into exactly 20 minutes.
As the sun sets, the luminosity gradually decreases.
The movie contains no sudden or abrupt stimuli, such as faces, aside from occasional passing cars in the village.
Slow and steady movement visible throughout are the waves, the lights flickering and the clouds drifting.
The movie is silent. 
```

Comment (oesteban) [reply README.md:51]: cc @acionca to confirm the description is appropriate :)

File: README.md [hunk 2]
```diff
@@ -47,6 +47,17 @@ The units reported here correspond to *Psychopy*'s "normalized units".
 4. Fixation: Blank trials consist of a fixation point presented at the center of the screen for 3s.
     The fixation point is built like in the gaze movement task.
 
+## Resting-state fMRI
+Resting-state fMRI was acquired while the participant watched a naturalistic scene depicting a calm landscape at sunset.
+The movie was captured by a static camera offering the view of the village of Mundaka, located on the northern coast of Spain, as shown in the image below.
+From the original recording of over four hours in length, we extracted 40 minutes, and increased the speed to compress them into exactly 20 minutes.
+As the sun sets, the luminosity gradually decreases.
+The movie contains no sudden or abrupt stimuli, such as faces, aside from occasional passing cars in the village.
```
Comment (acionca) [inline README.md:55]: If you really want to be detailed :)

```suggestion
The movie contains no sudden or abrupt stimuli, such as faces, aside from occasional passing cars in the village or one boat in the ocean.
```

**Review by acionca — COMMENTED** [review]
Looks good otherwise.

The movie has indeed been accelerated to match the 20 minutes of the fMRI sequence and to reduce the time where we see moving boats (or cars).

File: README.md (hunk 2 above)
Comment (celprov) [reply README.md]: How did you select which 40-minutes segment to keep? and why 40 minutes (and not 60 minutes compressed 3x e.g)?

**Comment by celprov** [comment]
@acionca How did you select which 40-minutes segment to keep? and why 40 minutes (and not 60 minutes compressed 3x e.g)? (just trying to make sure we are as detailed as possible in the task description and why we made those choices)

**Comment by celprov** [comment]
@acionca what do you think about the new paragraph incorporating your bullet points?

**Review by acionca — COMMENTED** [review]
I think that's more accurate - the movie has NOT been accelerated in the end :)

File: README.md [hunk 3]
```diff
@@ -50,11 +50,12 @@ The units reported here correspond to *Psychopy*'s "normalized units".
 ## Resting-state fMRI
 Resting-state fMRI was acquired while the participant watched a naturalistic scene depicting a calm landscape at sunset.
 The movie was captured by a static camera offering the view of the village of Mundaka, located on the northern coast of Spain, as shown in the image below.
-From the original recording of over four hours in length, we extracted 40 minutes, and increased the speed to compress them into exactly 20 minutes.
+From the original four-hour recording, we extracted a 40-minutes segment, which was accelerated to 20 minutes to fit the duration of the fMRI sequence.
+The segment was carefully selected to minimize distractions: the first part of the movie was cut out because a boat slowly moved in the right part of the frame and we didn't want it to become a point of focus for the participant. We also cut the end of the recording because the scene became too dark, which could induce sleep. Accelerating the selected segment further minimized the time that moving boats or cars were visible.
```
[the remark this answers is not in the document]
Comment (celprov) [reply README.md:54]: aaah the movie was not accelerated ?!? then we need to remove the last sentence "Accelerating the selected segment..." isn't it?

[the remark this answers is not in the document]
Comment (acionca) [reply README.md:54]: Indeed, I removed that - how does it look ?

[the remark this answers is not in the document]
Comment (celprov) [reply README.md:54]: Oh ok weird before refreshing that sentence was not cut out in your suggestion. All good, thanks :)

**Pull request merged**